routes/socket/routes.js

```diff
@@ -74,19 +74,21 @@ const gamesGarbageCollector = () => {
 	Object.keys(games).forEach(gameName => {
 		let toDelete = false;
 		const currentGame = games[gameName];
+		if (!currentGame) return;
+
 		const createdTimer =
-			currentGame &&
 			currentGame.general &&
 			currentGame.general.timeCreated &&
 			currentGame.gameState &&
 			!currentGame.gameState.isStarted &&
 			new Date(currentGame.general.timeCreated.getTime() + 600000);
```

Comment:
Just a pet peeve of mine but you use 1000 * 60 * numberOfMinutes everywhere else so this should be changed to be consistent with that.